pyproject.toml

```
@@ -61,3 +61,38 @@ version_file = "_icepyx_version.py"
 version_file_template = 'version = "{version}"'
 local_scheme = "node-and-date"
 fallback_version = "unknown"
+
+
+[tool.ruff.lint]
+select = [
+  "E",
+  "F",
```

Annotate what these single-letter rules are.

```suggestion
  "E",    # pycodestyle
  "F",    # pyflakes
```

---

Great call :)